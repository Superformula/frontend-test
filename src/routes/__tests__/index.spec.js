import React from 'react';
import { shallow } from 'enzyme';
import { Routes } from '../';

describe('Routes', () => {
  it('should render routes without failing', () => {
    const wrapper = shallow(<Routes />);
    expect(wrapper).toMatchSnapshot();
  });
});
