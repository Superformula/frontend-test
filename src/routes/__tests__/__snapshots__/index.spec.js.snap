// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should render routes without failing 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.lazy),
          "_init": [Function],
          "_payload": Object {
            "_result": [Function],
            "_status": -1,
          },
        }
      }
      exact={true}
      path="/restaurants"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.lazy),
          "_init": [Function],
          "_payload": Object {
            "_result": [Function],
            "_status": -1,
          },
        }
      }
      exact={true}
      path="/restaurants/:id"
    />
    <Redirect
      from="*"
      to="/restaurants"
    />
  </Switch>
</BrowserRouter>
`;
